data/hai/hai reveal 5 philosophers.txt

```diff
@@ -533,9 +642,12 @@ mission "Hai Reveal [B06-long] Boarded Smuggler"
 		government "Smuggler (Hai Trafficker)"
 	on offer
 		conversation
-			`As you maneuver to board this ship you send a message telling them that you know they're smugglers and demand their surrender. There is no response, but once you dock and open your side of the airlock, the door to the smuggler ship <origin> also opens partway and a Hai pokes their head through. The Hai's eyes are wide with fear but also determination as they shout at you through the docking tube, "This door is jammed, they're locking down the ship and going to blow it up!"`
-			`	In the background you hear a booming sound. "They're using the keystones, tell them they're using the keystones!"`
-			`	Inside your own ship you hear danger alarms suddenly blare into full volume and you have just enough time to nod a clear acknowledgment to the doomed Hai before your airlock slams shut with automatic safeties. A moment later, you hear a very large explosion and the sound of tearing metal as the docking tube is shredded and your ship rocks as it is forced away from the expanding debris of the smuggler ship. When you return to the controls it seems the smugglers signaled for help before the ship exploded. An escape pod is rapidly escaping and pirate signatures are on a swift approach. Apparently the smugglers care more about not being caught than about finishing their mission.`
+			`As you maneuver to board this ship you send a message telling them that you know they're smugglers and demand their surrender. There is no response, but once you dock and open your side of the airlock, the door to the smuggler ship <origin> also opens partway and a Hai pokes their head through. The Hai's eyes are wide with fear but also determination as they shout at you through the docking tube.`
+			choice
+				`	"Come on, you can do it! Just push a little harder!"`
+				`	"There should be an access panel above you! Maybe you can short-circuit the door?"`
+			`	The Hai says, "This door is jammed, they're locking down the ship and going to blow it up!" In the background you hear a booming sound. "They're using the keystones, tell them they're using the keystones!"`
+			`	Inside your ship you hear alarms blare into full volume and you have just enough time to nod an acknowledgment to the doomed Hai before the airlock's automatic safeties shut it. A moment later, you hear an explosion and the sound of tearing metal as the docking tube is shredded and your ship is forced away from the expanding debris of the smuggler ship. When you return to the controls it seems the smugglers signaled for help before the ship exploded. An escape pod is rapidly escaping and pirates are on a swift approach. Apparently the smugglers care more about not being caught than about finishing their mission.`
```

Comment:
```suggestion
			`	Inside your ship you hear alarms blare into full volume and you have just enough time to nod an acknowledgment to the doomed Hai before the airlock's automatic safeties shut it. A moment later, you hear an explosion and the sound of tearing metal as the docking tube is shredded and your ship is forced away from the expanding debris of the smuggler ship. When you return to the controls it seems the smugglers signaled for help before the ship exploded. An escape pod is flying away from the scene; apparently the smugglers care more about not being caught than about finishing their mission.`
```

Reply:
This removes the information that pirates are approaching.

Reply:
The sentence before didn't make sense, and we're told pirates are coming in the next paragraph.

Reply:
The next paragraph doesn't say the ships are pirates, though... but I'm not sure that it needs to.

Reply:
It was repetitive anyway.